### ISSUE TITLE
a11y: avatar group aria-label and i18n

### ISSUE THREAD

**Issue by web-padawan** [vaadin/vaadin-avatar#47]
From https://github.com/vaadin/vaadin-avatar/issues/10#issuecomment-656802750

- `anonymous` should be customizable (i18n)
- `<vaadin-avatar-group>`should get an additional `aria-label` with an i18n-able string like "Currently active users"

**Comment by knoobie** [comment]
Just an idea; add a count of current active users as variable. "Currently {{count}} active users"
